test/unit/render-util.test.js

```diff
@@ -13,3 +14,32 @@ describe('core/render/utils', () => {
     });
   });
 });
+
+describe('core/render/tpl', () => {
+  describe('remove html tag in tree', () => {
+    test('remove span and img', () => {
```

Comment:
Another minor nit-pick...

The regular expression used will remove *any* HTML tag, not just `<span>` and `<img>` tags. I think we can remove the nested `describe()` block and rename the `test()` block accordingly:

```js
describe('core/render/tpl', () => {
   test('remove html tags in tree', () => {
```